### DISCUSSION TITLE
Impact on date parsing due to Sunshine Protection Act?

### DISCUSSION THREAD

**Discussion by arahansen** [js-fns/date-fns#2999] [Q&A]
The US seems on its way to passing a bill (the [Sunshine Protection Act](https://www.reuters.com/world/us/us-senate-approves-bill-that-would-make-daylight-savings-time-permanent-2023-2022-03-15/)) which will make daylight savings time a permanent thing for all US timezones.

This might sound like a silly question, but what impact does this have (if any) on software systems that automatically switch over to DST/non-DST after a certain date? Does every system that handles dates need to be updated to account for this legislature? Does everything magically just work? Or do I have a misunderstanding of the impact here?

I'm not even sure if this is technically in the realm of date-fns to have an opinion on. Feel free to close if this is deemed irrelevant.

**Comment by tan75** [comment]
hello @arahansen 
thank you for your question - it's a good one.
From our side, we will not do anything with it - just need to wait for the browsers (or Node js) to update their time zone databases.
Once the dbs are updated, the changes will be visible in the results of the date-fns functions. 
Basically its up to the browsers or node how to handle these changes.
I am not sure how other systems handle this so I can only suppose, the idea is the same. 
Hope this helps!
